Cubical/HITs/Truncation/Base.agda

```diff
@@ -1,29 +1,34 @@
+{-
+
+A simpler definition of truncation ∥ A ∥ n from n ≥ -1
+
+Note that this uses the HoTT book's indexing, so it will be off
+ from `∥_∥_` in HITs.Truncation.Base by -2
+
+-}
 {-# OPTIONS --cubical --no-import-sorts --safe #-}
 module Cubical.HITs.Truncation.Base where
 
+open import Cubical.Data.NatMinusOne renaming (suc₋₁ to suc)
 open import Cubical.Foundations.Prelude
 open import Cubical.Foundations.HLevels
-open import Cubical.Data.Nat
-open import Cubical.Data.NatMinusOne
-open import Cubical.HITs.Nullification
 open import Cubical.HITs.Sn
+open import Cubical.Data.Nat.Base renaming (suc to sucℕ)
+open import Cubical.Data.Unit.Base
+open import Cubical.Data.Empty
 
--- For the hub-and-spoke construction discussed in the HoTT book, which doesn't work in the base case
---  of contractibility, see `HITs.Truncation.FromNegOne`. The definition of truncation here contains
---  two more constructors which are redundant when n ≥ 1 but give contractibility when n = 0.
+S' : ℕ₋₁ → Type₀
+S' neg1 = ⊥
+S' (ℕ→ℕ₋₁ n) = S₊ n
 
--- data hLevelTrunc {ℓ} (n : HLevel) (A : Type ℓ) : Type (ℓ-max ℓ ℓ') where
---   -- the hub-and-spoke definition in `Truncation.FromNegOne`
---   ∣_∣ : A → hLevelTrunc n A
---   hub   : (f : S (-1+ n) → hLevelTrunc n A) → hLevelTrunc n A
---   spoke : (f : S (-1+ n) → hLevelTrunc n A) (s : S) → hub f ≡ f s
---   -- two additional constructors needed to ensure that hLevelTrunc 0 A is contractible
---   ≡hub   : ∀ {x y} (p : S (-1+ n) → x ≡ y) → x ≡ y
---   ≡spoke : ∀ {x y} (p : S (-1+ n) → x ≡ y) (s : S (-1+ n)) → ≡hub p ≡ p s
+data HubAndSpoke {ℓ} (A : Type ℓ) (n : ℕ₋₁) : Type ℓ where
+  ∣_∣ : A  → HubAndSpoke A n
+  hub : (f : S' (suc n) → HubAndSpoke A n) → HubAndSpoke A n
+  spoke : (f : S' (suc n) → HubAndSpoke A n) (x : S' (suc n)) → hub f ≡ f x
```

Comment:
So what it necessary to introduce `S'` and use `suc n` here? It really seems like you can simplify it

Reply:
Sorry for the delay, I got distracted by other stuff as usual:-) It's fixed now.